test/parallel/test-assert-objects.js

```diff
@@ -0,0 +1,605 @@
+          expected: null,
+        },
+      ].forEach(({ description, actual, expected }) => {
+        it(description, () => {
+          assert.throws(() => assert.partialDeepEqual(actual, expected), Error);
+        });
+      });
+    });
+
+    describe('does not throw an error', () => {
+      [
+        {
+          description: 'compares two objects with Buffers',
+          actual: { buf: Buffer.from('Node.js') },
+          expected: { buf: Buffer.from('Node.js') },
+        },
+        {
+          description: 'compares two identical primitives, string',
+          actual: 'foo',
+          expected: 'foo',
+        },
+
+        {
+          description: 'compares two identical primitives, number',
+          actual: 1,
+          expected: 1,
+        },
+
+        {
+          description: 'compares two identical primitives, bigint',
+          actual: 1n,
+          expected: 1n,
+        },
+
+        {
+          description: 'compares two non identical simple objects',
+          actual: { a: 1, b: 'foo', c: '1' },
+          expected: { a: 1, c: 1 },
+        },
+        {
+          description: 'compares two similar objects with type coercion',
+          actual: { a: 1, b: '2' },
+          expected: { a: 1, b: 2 },
+        },
+        {
+          description:
+            'compares two objects with nested objects and type coercion',
+          actual: { level1: { level2: { level3: '42' } } },
+          expected: { level1: { level2: { level3: 42 } } },
+        },
+        {
+          description: 'compares two objects with circular references',
+          actual: createCircularObject(),
+          expected: createCircularObject(),
+        },
+        {
+          description: 'compares two arrays with type coercion',
+          actual: [1, '2', true],
+          expected: [1, 2, 1],
+        },
+        {
+          description: 'compares two objects with numeric string and number',
+          actual: { a: '100' },
+          expected: { a: 100 },
+        },
+        {
+          description:
+            'compares two objects with boolean and numeric representations',
+          actual: { a: 1, b: 0 },
+          expected: { a: true, b: false },
+        },
+        {
+          description:
+            'compares two objects with undefined and missing properties',
+          actual: { a: undefined },
+          expected: {},
+        },
+        {
+          description: 'compares one subset object with another',
+          actual: { a: 1, b: 2, c: 3 },
+          expected: { b: '2' },
+        },
+        {
+          description: 'compares one subset array with another',
+          actual: [true],
+          expected: [1],
+        },
+      ].forEach(({ description, actual, expected }) => {
+        it(description, () => {
+          assert.partialDeepEqual(actual, expected);
+        });
+      });
+    });
+  });
+
+  describe('includes', () => {
+    describe('throws an error', () => {
```

Comment:
Can you add the following cases:
1. `expected` is not a valid value
2. `actual` array is being compared to `expected` string

Reply:
1. `expected` is `any`, added a test for when `expected` is missing
2: 👍